UnitedIncome/components
# [APP-5044] SimpleSummary TypeScript

### Description
> Please provide a brief description of your changes below.

Converts the `SimpleSummary` card to Typescript.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* There should be no changes.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

N/A.

### Comments
> Any additional comments that will aid with the review process.

Had to fix up a few references to components that use this one in order to get it to compile correctly. 

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5044)
